Execution of processAllLogs_new.py started at 2020-07-09 05:00:02.205770
Now processing logs0
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes0/2020-07-08/13
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes0/2020-07-08/14
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes0/2020-07-08/15
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes0/2020-07-08/16
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes0/2020-07-08/17
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes0/2020-07-08/18
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes0/2020-07-08/19
making directory: /h
Execution of processAllLogs_new.py started at 2020-07-10 05:00:03.757422
Now processing logs0
making directory: /h
Execution of processAllLogs_new.py started at 2020-07-11 05:00:03.291421
Now processing logs0
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes0/2020-07-10/13
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes0/2020-07-10/17
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes0/2020-07-10/19
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes0/2020-07-10/20
making directory: /h
Execution of processAllLogs_new.py started at 2020-07-12 05:00:02.045998
Now processing logs0
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes0/2020-07-11/12
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes0/2020-07-11/13
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes0/2020-07-11/14
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes0/2020-07-11/15
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes0/2020-07-11/16
Execution of processAllLogs_new.py started at 2020-07-13 05:00:02.441420
Now processing logs0
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes0/2020-07-12/20
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes0/2020-07-11/11
Execution of processAllLogs_new.py started at 2020-07-14 05:00:01.664945
Now processing logs0
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes0/2020-07-13/20
Execution of processAllLogs_new.py started at 2020-07-14 16:01:02.339468
Now processing logs0
making directory: /h
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_16:45:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_16:50:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_16:55:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_17:00:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_11:30:03_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_11:35:03_0.log
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes0/2020-07-14/12
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes0/2020-07-14/13
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes0/2020-07-14/14
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes0/2020-07-14/15
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes1/2020-07-14/15
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-14 16:39:57.772643
Execution of processAllLogs_new.py started at 2020-07-14 17:01:01.978103
Now processing logs0
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_17:00:04_0.log
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes0/2020-07-14/16
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_16:55:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_11:30:03_0.log
making directory: /h
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_11:35:03_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_16:45:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_16:50:04_0.log
Now processing logs1
making directory: /h
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes1/2020-07-14/16
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-14 17:08:00.852286
Execution of processAllLogs_new.py started at 2020-07-14 18:01:01.937295
Now processing logs0
making directory: /h
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_16:45:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_16:50:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_16:55:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_17:00:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_11:30:03_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_11:35:03_0.log
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes0/2020-07-14/17
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-07-14_16:50:04_1.log
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes1/2020-07-14/17
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-14 18:12:31.494840
Execution of processAllLogs_new.py started at 2020-07-14 19:01:01.663457
Now processing logs0
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_16:50:04_0.log
making directory: /h
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_11:30:03_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_17:45:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_17:50:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_11:35:03_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_16:55:04_0.log
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes0/2020-07-14/18
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_17:00:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_17:55:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_16:45:04_0.log
Now processing logs1
making directory: /h
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-07-14_16:50:04_1.log
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes1/2020-07-14/18
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-14 19:09:31.431805
Execution of processAllLogs_new.py started at 2020-07-14 20:01:01.531518
Now processing logs0
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_16:50:04_0.log
making directory: /h
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_11:30:03_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_17:45:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_17:50:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_11:35:03_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_16:55:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_17:00:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_17:55:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_16:45:04_0.log
Now processing logs1
making directory: /h
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-07-14_16:50:04_1.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-14 20:08:03.923849
Execution of processAllLogs_new.py started at 2020-07-14 21:01:01.728527
Now processing logs0
making directory: /h
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_16:45:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_16:50:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_16:55:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_17:00:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_11:30:03_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_11:35:03_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_17:45:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_17:50:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_17:55:04_0.log
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes0/2020-07-14/20
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-07-14_16:50:04_1.log
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes1/2020-07-14/20
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-14 21:10:04.199276
Execution of processAllLogs_new.py started at 2020-07-14 22:01:01.706854
Now processing logs0
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_16:50:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_17:50:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_17:45:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_17:00:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_16:45:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_17:55:04_0.log
making directory: /h
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_11:30:03_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_11:35:03_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_16:55:04_0.log
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-07-14_16:50:04_1.log
making directory: /h
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-14 22:08:22.673420
Execution of processAllLogs_new.py started at 2020-07-14 23:01:02.449362
Now processing logs0
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_16:50:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_17:50:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_17:45:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_17:00:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_16:45:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_17:55:04_0.log
making directory: /h
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_11:30:03_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_11:35:03_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_16:55:04_0.log
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-07-14_16:50:04_1.log
making directory: /h
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-14 23:08:22.584078
Execution of processAllLogs_new.py started at 2020-07-15 00:01:02.074588
Now processing logs0
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_16:50:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_17:50:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_17:45:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_17:00:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_16:45:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_17:55:04_0.log
making directory: /h
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_11:30:03_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_11:35:03_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_16:55:04_0.log
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-07-14_16:50:04_1.log
making directory: /h
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-15 00:08:46.931714
Execution of processAllLogs_new.py started at 2020-07-15 01:01:01.957803
Now processing logs0
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_16:50:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_17:50:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_17:45:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_17:00:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_16:45:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_17:55:04_0.log
making directory: /h
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_11:30:03_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_11:35:03_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_16:55:04_0.log
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-07-14_16:50:04_1.log
making directory: /h
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-15 01:14:47.677055
Execution of processAllLogs_new.py started at 2020-07-15 02:01:01.881450
Now processing logs0
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_16:50:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_17:50:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_17:45:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_17:00:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_16:45:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_17:55:04_0.log
making directory: /h
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_11:30:03_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_11:35:03_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_16:55:04_0.log
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-07-14_16:50:04_1.log
making directory: /h
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-15 02:14:57.915978
Execution of processAllLogs_new.py started at 2020-07-15 03:01:02.330280
Now processing logs0
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_16:50:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_17:50:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_17:45:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_17:00:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_16:45:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_17:55:04_0.log
making directory: /h
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_11:30:03_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_11:35:03_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_16:55:04_0.log
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-07-14_16:50:04_1.log
making directory: /h
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-15 03:15:02.927684
Execution of processAllLogs_new.py started at 2020-07-15 04:01:02.019703
Now processing logs0
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_16:50:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_17:50:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_17:45:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_17:00:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_16:45:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_17:55:04_0.log
making directory: /h
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_11:30:03_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_11:35:03_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_16:55:04_0.log
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-07-14_16:50:04_1.log
making directory: /h
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-15 04:14:38.039635
Execution of processAllLogs_new.py started at 2020-07-15 05:01:01.849622
Now processing logs0
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_16:50:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_17:50:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_17:45:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_17:00:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_16:45:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_17:55:04_0.log
making directory: /h
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_11:30:03_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_11:35:03_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_16:55:04_0.log
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-07-14_16:50:04_1.log
making directory: /h
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-15 05:14:32.647740
Execution of processAllLogs_new.py started at 2020-07-15 06:01:02.042753
Now processing logs0
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_16:50:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_17:50:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_17:45:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_17:00:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_16:45:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_17:55:04_0.log
making directory: /h
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_11:30:03_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_11:35:03_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_16:55:04_0.log
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-07-14_16:50:04_1.log
making directory: /h
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-15 06:15:18.855002
Execution of processAllLogs_new.py started at 2020-07-15 07:01:01.856551
Now processing logs0
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_16:50:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_17:50:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_17:45:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_17:00:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_16:45:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_17:55:04_0.log
making directory: /h
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_11:30:03_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_11:35:03_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_16:55:04_0.log
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-07-14_16:50:04_1.log
making directory: /h
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-15 07:14:47.468506
Execution of processAllLogs_new.py started at 2020-07-15 08:01:02.057975
Now processing logs0
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes0/2020-07-15/07
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_16:50:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_17:50:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_17:45:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_17:00:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_16:45:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_17:55:04_0.log
making directory: /h
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_11:30:03_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_11:35:03_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_16:55:04_0.log
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-07-14_16:50:04_1.log
making directory: /h
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes1/2020-07-15/07
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-15 08:18:54.573715
Execution of processAllLogs_new.py started at 2020-07-15 09:01:02.177080
Now processing logs0
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_16:50:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_17:50:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_17:45:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_17:00:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_16:45:04_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_17:55:04_0.log
making directory: /h
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_11:30:03_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-14_11:35:03_0.log
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-09_16:55:04_0.log
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-07-14_16:50:04_1.log
making directory: /h
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-15 09:15:03.860870
Execution of processAllLogs_new.py started at 2020-07-15 11:01:02.393984
Now processing logs0
Execution of processAllLogs_new.py started at 2020-07-15 12:01:01.520058
Now processing logs0
Execution of processAllLogs_new.py started at 2020-07-15 13:01:02.317255
Now processing logs0
making directory: /h
Execution of processAllLogs_new.py started at 2020-07-15 14:01:02.327757
Now processing logs0
Execution of processAllLogs_new.py started at 2020-07-15 15:01:02.142577
Now processing logs0
Traceback (most recent call last):
  File "/home/ardelalegre/CSE4223-ODAS/preprocessing/python/processAllLogs_new.py", line 107, in <module>
    temp = shutil.move(log,destination) # new
  File "/usr/lib/python2.7/shutil.py", line 292, in move
    raise Error, "Destination path '%s' already exists" % real_dst
shutil.Error: Destination path '/home/ardelalegre/google-drive/ODAS/logs0/SST/Processed/cSST_2020-07-14_21:55:03_0.log' already exists
Execution of processAllLogs_new.py started at 2020-07-15 16:01:02.111438
Now processing logs0
Traceback (most recent call last):
  File "/home/ardelalegre/CSE4223-ODAS/preprocessing/python/processAllLogs_new.py", line 107, in <module>
    temp = shutil.move(log,destination) # new
  File "/usr/lib/python2.7/shutil.py", line 292, in move
    raise Error, "Destination path '%s' already exists" % real_dst
shutil.Error: Destination path '/home/ardelalegre/google-drive/ODAS/logs0/SST/Processed/cSST_2020-07-14_21:55:03_0.log' already exists
Execution of processAllLogs_new.py started at 2020-07-15 17:01:01.597728
Now processing logs0
Traceback (most recent call last):
  File "/home/ardelalegre/CSE4223-ODAS/preprocessing/python/processAllLogs_new.py", line 107, in <module>
    temp = shutil.move(log,destination) # new
  File "/usr/lib/python2.7/shutil.py", line 292, in move
    raise Error, "Destination path '%s' already exists" % real_dst
shutil.Error: Destination path '/home/ardelalegre/google-drive/ODAS/logs0/SST/Processed/cSST_2020-07-08_04:45:03_0.log' already exists
Execution of processAllLogs_new.py started at 2020-07-15 18:01:02.466763
Now processing logs0
Traceback (most recent call last):
  File "/home/ardelalegre/CSE4223-ODAS/preprocessing/python/processAllLogs_new.py", line 107, in <module>
    temp = shutil.move(log,destination) # new
  File "/usr/lib/python2.7/shutil.py", line 292, in move
    raise Error, "Destination path '%s' already exists" % real_dst
shutil.Error: Destination path '/home/ardelalegre/google-drive/ODAS/logs0/SST/Processed/cSST_2020-07-12_04:45:03_0.log' already exists
Execution of processAllLogs_new.py started at 2020-07-15 19:01:02.349883
Now processing logs0
Traceback (most recent call last):
  File "/home/ardelalegre/CSE4223-ODAS/preprocessing/python/processAllLogs_new.py", line 107, in <module>
    temp = shutil.move(log,destination) # new
  File "/usr/lib/python2.7/shutil.py", line 292, in move
    raise Error, "Destination path '%s' already exists" % real_dst
shutil.Error: Destination path '/home/ardelalegre/google-drive/ODAS/logs0/SST/Processed/cSST_2020-07-12_04:45:03_0.log' already exists
Execution of processAllLogs_new.py started at 2020-07-15 20:01:02.261838
Now processing logs0
Traceback (most recent call last):
  File "/home/ardelalegre/CSE4223-ODAS/preprocessing/python/processAllLogs_new.py", line 107, in <module>
    temp = shutil.move(log,destination) # new
  File "/usr/lib/python2.7/shutil.py", line 292, in move
    raise Error, "Destination path '%s' already exists" % real_dst
shutil.Error: Destination path '/home/ardelalegre/google-drive/ODAS/logs0/SST/Processed/cSST_2020-07-12_04:45:03_0.log' already exists
Execution of processAllLogs_new.py started at 2020-07-15 21:01:01.728535
Now processing logs0
Traceback (most recent call last):
  File "/home/ardelalegre/CSE4223-ODAS/preprocessing/python/processAllLogs_new.py", line 107, in <module>
    temp = shutil.move(log,destination) # new
  File "/usr/lib/python2.7/shutil.py", line 292, in move
    raise Error, "Destination path '%s' already exists" % real_dst
shutil.Error: Destination path '/home/ardelalegre/google-drive/ODAS/logs0/SST/Processed/cSST_2020-07-12_04:45:03_0.log' already exists
Execution of processAllLogs_new.py started at 2020-07-15 22:01:01.838215
Now processing logs0
Traceback (most recent call last):
  File "/home/ardelalegre/CSE4223-ODAS/preprocessing/python/processAllLogs_new.py", line 107, in <module>
    temp = shutil.move(log,destination) # new
  File "/usr/lib/python2.7/shutil.py", line 292, in move
    raise Error, "Destination path '%s' already exists" % real_dst
shutil.Error: Destination path '/home/ardelalegre/google-drive/ODAS/logs0/SST/Processed/cSST_2020-07-12_04:45:03_0.log' already exists
Execution of processAllLogs_new.py started at 2020-07-15 23:01:02.098389
Now processing logs0
Traceback (most recent call last):
  File "/home/ardelalegre/CSE4223-ODAS/preprocessing/python/processAllLogs_new.py", line 107, in <module>
    temp = shutil.move(log,destination) # new
  File "/usr/lib/python2.7/shutil.py", line 292, in move
    raise Error, "Destination path '%s' already exists" % real_dst
shutil.Error: Destination path '/home/ardelalegre/google-drive/ODAS/logs0/SST/Processed/cSST_2020-07-12_04:45:03_0.log' already exists
Execution of processAllLogs_new.py started at 2020-07-16 00:01:01.760015
Now processing logs0
Traceback (most recent call last):
  File "/home/ardelalegre/CSE4223-ODAS/preprocessing/python/processAllLogs_new.py", line 107, in <module>
    temp = shutil.move(log,destination) # new
  File "/usr/lib/python2.7/shutil.py", line 292, in move
    raise Error, "Destination path '%s' already exists" % real_dst
shutil.Error: Destination path '/home/ardelalegre/google-drive/ODAS/logs0/SST/Processed/cSST_2020-07-12_04:45:03_0.log' already exists
Execution of processAllLogs_new.py started at 2020-07-16 01:01:01.617874
Now processing logs0
Traceback (most recent call last):
  File "/home/ardelalegre/CSE4223-ODAS/preprocessing/python/processAllLogs_new.py", line 107, in <module>
    temp = shutil.move(log,destination) # new
  File "/usr/lib/python2.7/shutil.py", line 292, in move
    raise Error, "Destination path '%s' already exists" % real_dst
shutil.Error: Destination path '/home/ardelalegre/google-drive/ODAS/logs0/SST/Processed/cSST_2020-07-12_04:45:03_0.log' already exists
Execution of processAllLogs_new.py started at 2020-07-16 02:01:02.248267
Now processing logs0
Traceback (most recent call last):
  File "/home/ardelalegre/CSE4223-ODAS/preprocessing/python/processAllLogs_new.py", line 107, in <module>
    temp = shutil.move(log,destination) # new
  File "/usr/lib/python2.7/shutil.py", line 292, in move
    raise Error, "Destination path '%s' already exists" % real_dst
shutil.Error: Destination path '/home/ardelalegre/google-drive/ODAS/logs0/SST/Processed/cSST_2020-07-12_04:45:03_0.log' already exists
Execution of processAllLogs_new.py started at 2020-07-16 03:01:02.033556
Now processing logs0
Traceback (most recent call last):
  File "/home/ardelalegre/CSE4223-ODAS/preprocessing/python/processAllLogs_new.py", line 107, in <module>
    temp = shutil.move(log,destination) # new
  File "/usr/lib/python2.7/shutil.py", line 292, in move
    raise Error, "Destination path '%s' already exists" % real_dst
shutil.Error: Destination path '/home/ardelalegre/google-drive/ODAS/logs0/SST/Processed/cSST_2020-07-12_04:45:03_0.log' already exists
Execution of processAllLogs_new.py started at 2020-07-16 04:01:02.388727
Now processing logs0
Traceback (most recent call last):
  File "/home/ardelalegre/CSE4223-ODAS/preprocessing/python/processAllLogs_new.py", line 107, in <module>
    temp = shutil.move(log,destination) # new
  File "/usr/lib/python2.7/shutil.py", line 292, in move
    raise Error, "Destination path '%s' already exists" % real_dst
shutil.Error: Destination path '/home/ardelalegre/google-drive/ODAS/logs0/SST/Processed/cSST_2020-07-12_04:45:03_0.log' already exists
Execution of processAllLogs_new.py started at 2020-07-16 05:01:01.532100
Now processing logs0
Traceback (most recent call last):
  File "/home/ardelalegre/CSE4223-ODAS/preprocessing/python/processAllLogs_new.py", line 107, in <module>
    temp = shutil.move(log,destination) # new
  File "/usr/lib/python2.7/shutil.py", line 292, in move
    raise Error, "Destination path '%s' already exists" % real_dst
shutil.Error: Destination path '/home/ardelalegre/google-drive/ODAS/logs0/SST/Processed/cSST_2020-07-12_04:45:03_0.log' already exists
Execution of processAllLogs_new.py started at 2020-07-16 06:01:01.687614
Now processing logs0
Traceback (most recent call last):
  File "/home/ardelalegre/CSE4223-ODAS/preprocessing/python/processAllLogs_new.py", line 107, in <module>
    temp = shutil.move(log,destination) # new
  File "/usr/lib/python2.7/shutil.py", line 292, in move
    raise Error, "Destination path '%s' already exists" % real_dst
shutil.Error: Destination path '/home/ardelalegre/google-drive/ODAS/logs0/SST/Processed/cSST_2020-07-12_04:45:03_0.log' already exists
Execution of processAllLogs_new.py started at 2020-07-16 07:01:02.295131
Now processing logs0
Traceback (most recent call last):
  File "/home/ardelalegre/CSE4223-ODAS/preprocessing/python/processAllLogs_new.py", line 107, in <module>
    temp = shutil.move(log,destination) # new
  File "/usr/lib/python2.7/shutil.py", line 292, in move
    raise Error, "Destination path '%s' already exists" % real_dst
shutil.Error: Destination path '/home/ardelalegre/google-drive/ODAS/logs0/SST/Processed/cSST_2020-07-12_04:45:03_0.log' already exists
Execution of processAllLogs_new.py started at 2020-07-16 08:01:01.719988
Now processing logs0
Traceback (most recent call last):
  File "/home/ardelalegre/CSE4223-ODAS/preprocessing/python/processAllLogs_new.py", line 105, in <module>
    firstline = f.readline()
IOError: [Errno 5] Input/output error
Execution of processAllLogs_new.py started at 2020-07-16 09:01:01.634932
Now processing logs0
Traceback (most recent call last):
  File "/home/ardelalegre/CSE4223-ODAS/preprocessing/python/processAllLogs_new.py", line 105, in <module>
    firstline = f.readline()
IOError: [Errno 5] Input/output error
/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-16 10:01:01.765451
Now processing logs0
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes0/2020-07-15/14
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes0/2020-07-15/15
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes0/2020-07-15/16
making directory: /h
Traceback (most recent call last):
  File "/home/ardelalegre/CSE4223-ODAS/preprocessing/python/processAllLogs_new.py", line 107, in <module>
    temp = shutil.move(log,destination) # new
  File "/usr/lib/python2.7/shutil.py", line 292, in move
    raise Error, "Destination path '%s' already exists" % real_dst
shutil.Error: Destination path '/home/ardelalegre/google-drive/ODAS/logs0/SST/Processed/cSST_2020-07-08_04:45:03_0.log' already exists
Execution of processAllLogs_new.py started at 2020-07-16 11:01:01.476227
Now processing logs0
Traceback (most recent call last):
  File "/home/ardelalegre/CSE4223-ODAS/preprocessing/python/processAllLogs_new.py", line 107, in <module>
    temp = shutil.move(log,destination) # new
  File "/usr/lib/python2.7/shutil.py", line 292, in move
    raise Error, "Destination path '%s' already exists" % real_dst
shutil.Error: Destination path '/home/ardelalegre/google-drive/ODAS/logs0/SST/Processed/cSST_2020-07-10_04:45:03_0.log' already exists
Execution of processAllLogs_new.py started at 2020-07-16 12:01:01.997755
Now processing logs0
Traceback (most recent call last):
  File "/home/ardelalegre/CSE4223-ODAS/preprocessing/python/processAllLogs_new.py", line 107, in <module>
    temp = shutil.move(log,destination) # new
  File "/usr/lib/python2.7/shutil.py", line 292, in move
    raise Error, "Destination path '%s' already exists" % real_dst
shutil.Error: Destination path '/home/ardelalegre/google-drive/ODAS/logs0/SST/Processed/cSST_2020-07-10_04:45:03_0.log' already exists
Execution of processAllLogs_new.py started at 2020-07-16 13:01:01.638961
Now processing logs0
Traceback (most recent call last):
  File "/home/ardelalegre/CSE4223-ODAS/preprocessing/python/processAllLogs_new.py", line 107, in <module>
    temp = shutil.move(log,destination) # new
  File "/usr/lib/python2.7/shutil.py", line 292, in move
    raise Error, "Destination path '%s' already exists" % real_dst
shutil.Error: Destination path '/home/ardelalegre/google-drive/ODAS/logs0/SST/Processed/cSST_2020-07-10_04:45:03_0.log' already exists
Execution of processAllLogs_new.py started at 2020-07-16 14:01:02.413279
Now processing logs0
Traceback (most recent call last):
  File "/home/ardelalegre/CSE4223-ODAS/preprocessing/python/processAllLogs_new.py", line 107, in <module>
    temp = shutil.move(log,destination) # new
  File "/usr/lib/python2.7/shutil.py", line 292, in move
    raise Error, "Destination path '%s' already exists" % real_dst
shutil.Error: Destination path '/home/ardelalegre/google-drive/ODAS/logs0/SST/Processed/cSST_2020-07-10_04:45:03_0.log' already exists
Execution of processAllLogs_new.py started at 2020-07-16 15:01:01.740056
Now processing logs0
Traceback (most recent call last):
  File "/home/ardelalegre/CSE4223-ODAS/preprocessing/python/processAllLogs_new.py", line 107, in <module>
    temp = shutil.move(log,destination) # new
  File "/usr/lib/python2.7/shutil.py", line 292, in move
    raise Error, "Destination path '%s' already exists" % real_dst
shutil.Error: Destination path '/home/ardelalegre/google-drive/ODAS/logs0/SST/Processed/cSST_2020-07-14_18:55:04_0.log' already exists
Execution of processAllLogs_new.py started at 2020-07-16 16:01:02.816723
Now processing logs0
Traceback (most recent call last):
  File "/home/ardelalegre/CSE4223-ODAS/preprocessing/python/processAllLogs_new.py", line 107, in <module>
    temp = shutil.move(log,destination) # new
  File "/usr/lib/python2.7/shutil.py", line 292, in move
    raise Error, "Destination path '%s' already exists" % real_dst
shutil.Error: Destination path '/home/ardelalegre/google-drive/ODAS/logs0/SST/Processed/cSST_2020-07-14_22:55:03_0.log' already exists
Execution of processAllLogs_new.py started at 2020-07-16 17:01:02.153825
Now processing logs0
Traceback (most recent call last):
  File "/home/ardelalegre/CSE4223-ODAS/preprocessing/python/processAllLogs_new.py", line 107, in <module>
    temp = shutil.move(log,destination) # new
  File "/usr/lib/python2.7/shutil.py", line 292, in move
    raise Error, "Destination path '%s' already exists" % real_dst
shutil.Error: Destination path '/home/ardelalegre/google-drive/ODAS/logs0/SST/Processed/cSST_2020-07-15_06:50:03_0.log' already exists
/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-16 19:01:01.600591
Now processing logs0
making directory: /h
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes0/2020-07-16/18
Now processing logs1
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes1/2020-07-16/18
Traceback (most recent call last):
  File "/home/ardelalegre/CSE4223-ODAS/preprocessing/python/processAllLogs_new.py", line 104, in <module>
    with open(log, 'r') as f:
IOError: [Errno 2] No such file or directory: '/home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-07-15_14:15:04_1.log'
/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-16 20:01:02.358004
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
Traceback (most recent call last):
  File "/home/ardelalegre/CSE4223-ODAS/preprocessing/python/processAllLogs_new.py", line 104, in <module>
    with open(log, 'r') as f:
IOError: [Errno 2] No such file or directory: '/home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-07-16_16:00:04_1.log'
/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-16 21:01:02.523336
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Traceback (most recent call last):
  File "/home/ardelalegre/CSE4223-ODAS/preprocessing/python/processAllLogs_new.py", line 104, in <module>
    with open(log, 'r') as f:
IOError: [Errno 2] No such file or directory: '/home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-07-16_07:30:03_1.log'
/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-16 22:01:01.581287
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes2/2020-07-16/18
Traceback (most recent call last):
  File "/home/ardelalegre/CSE4223-ODAS/preprocessing/python/processAllLogs_new.py", line 104, in <module>
    with open(log, 'r') as f:
IOError: [Errno 2] No such file or directory: '/home/ardelalegre/google-drive/ODAS/logs2/SST/cSST_2020-07-16_17:10:04_2.log'
/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-16 18:01:02.376926
Now processing logs0
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes0/2020-07-16/16
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes0/2020-07-16/14
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes0/2020-07-16/17
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes0/2020-07-16/15
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes0/2020-07-16/13
making directory: /h
Now processing logs1
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes1/2020-07-16/13
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes1/2020-07-15/15
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes1/2020-07-16/14
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes1/2020-07-15/16
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes1/2020-07-15/14
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes1/2020-07-16/15
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes1/2020-07-16/11
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes1/2020-07-16/00
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes1/2020-07-16/17
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes1/2020-07-16/16
making directory: /h
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes1/2020-07-15/17
Now processing logs2
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes2/2020-07-16/15
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes2/2020-07-15/15
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes2/2020-07-15/16
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes2/2020-07-16/16
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes2/2020-07-15/14
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes2/2020-07-16/14
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes2/2020-07-16/13
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes2/2020-07-16/17
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes2/2020-07-16/05
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes2/2020-07-15/17
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes2/2020-07-16/11
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-16 23:21:39.056289


Execution of processAllLogs_new.py started at 2020-07-16 23:01:01.670441
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
Traceback (most recent call last):
  File "/home/ardelalegre/CSE4223-ODAS/preprocessing/python/processAllLogs_new.py", line 104, in <module>
    with open(log, 'r') as f:
IOError: [Errno 2] No such file or directory: '/home/ardelalegre/google-drive/ODAS/logs2/SST/cSST_2020-07-15_16:05:05_2.log'
/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-17 00:01:01.642447
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-17 00:14:00.592023


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-17 01:01:02.288439
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-17 01:13:35.680951


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-17 02:01:02.421884
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-17 02:13:29.017165


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-17 03:01:01.655828
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-17 03:13:33.083224


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-17 04:01:01.746796
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-17 04:13:26.364996


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-17 05:01:02.123989
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-17 05:13:34.709282


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-17 06:01:02.408680
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-17 06:22:13.660060


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-17 07:01:02.351126
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-17 07:19:13.373686


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-17 08:01:02.140224
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes2/2020-07-17/07
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-17 08:28:32.545831


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-17 09:01:02.195456
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-17 09:27:01.580611


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-17 10:01:02.320494
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-17 10:13:59.033705


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-17 11:01:01.790789
Now processing logs0
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes0/2020-07-17/10
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes1/2020-07-17/10
making directory: /h
Now processing logs2
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes2/2020-07-17/10
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-17 11:14:17.988871


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-17 12:01:01.723657
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes1/2020-07-17/11
making directory: /h
Now processing logs2
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes2/2020-07-17/11
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-17 12:16:27.750095


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-17 13:01:02.418068
Now processing logs0
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes0/2020-07-17/12
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes1/2020-07-17/12
Now processing logs2
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes2/2020-07-17/12
Now processing logs3
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes3/2020-07-17/12
Execution of processAllLogs_new.py ended at 2020-07-17 13:58:44.682137


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-17 14:01:02.322991
Now processing logs0
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes0/2020-07-17/13
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes1/2020-07-17/13
making directory: /h
Now processing logs2
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes2/2020-07-17/13
Now processing logs3
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes3/2020-07-17/13
Execution of processAllLogs_new.py ended at 2020-07-17 15:35:19.530873


Execution of processAllLogs_new.py started at 2020-07-17 15:01:01.543030
Now processing logs0
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes0/2020-07-17/14
Could not process file: /home/ardelalegre/google-drive/ODAS/logs0/SST/cSST_2020-07-17_13:55:04_0.log
making directory: /h
Now processing logs1
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes1/2020-07-17/14
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes2/2020-07-17/14
Traceback (most recent call last):
  File "/home/ardelalegre/CSE4223-ODAS/preprocessing/python/processAllLogs_new.py", line 104, in <module>
    with open(log, 'r') as f:
IOError: [Errno 2] No such file or directory: '/home/ardelalegre/google-drive/ODAS/logs2/SST/cSST_2020-07-17_13:35:04_2.log'
/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-17 16:01:02.412744
Now processing logs0
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes0/2020-07-17/15
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes1/2020-07-17/15
Now processing logs2
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes2/2020-07-17/15
Now processing logs3
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes3/2020-07-17/14
Execution of processAllLogs_new.py ended at 2020-07-17 16:31:03.782930


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-17 17:01:02.501763
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
Now processing logs3
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes3/2020-07-17/16
Execution of processAllLogs_new.py ended at 2020-07-17 17:52:56.498795


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-17 18:01:02.012027
Now processing logs0
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes0/2020-07-17/17
making directory: /h
Now processing logs1
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes1/2020-07-17/17
making directory: /h
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
Now processing logs2
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes2/2020-07-17/17
Now processing logs3
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes3/2020-07-17/17
Execution of processAllLogs_new.py ended at 2020-07-17 18:36:16.181854


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-17 19:01:01.912751
Now processing logs0
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes0/2020-07-17/18
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes1/2020-07-17/18
Now processing logs2
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes2/2020-07-17/18
Now processing logs3
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes3/2020-07-17/18
Execution of processAllLogs_new.py ended at 2020-07-17 19:36:34.489550


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-17 20:01:02.174640
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-17 20:18:07.538400


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-17 21:01:02.252456
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-17 21:18:00.845155


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-17 22:01:01.624936
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-17 22:18:07.451913


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-17 23:01:02.134661
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-17 23:18:35.536034


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-18 00:01:02.225685
Now processing logs0
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes0/2020-07-17/23
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-18 00:17:57.779086


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-18 01:01:02.607317
Now processing logs0
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes0/2020-07-18/00
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-18 01:18:12.493968


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-18 02:01:02.183158
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-18 02:17:44.478861


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-18 03:01:02.128067
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-18 03:17:52.150873


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-18 04:01:01.840584
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-18 04:17:55.427177


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-18 05:01:02.214726
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-18 05:17:35.918600


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-18 06:01:01.622129
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-18 06:33:55.514024


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-18 07:01:02.198032
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-18 07:25:18.858364


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-18 08:01:01.608924
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-18 08:28:32.719262


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-18 09:01:02.403166
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-18 09:17:08.113687


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-18 10:01:01.777867
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-18 10:16:43.776687


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-18 11:01:02.470413
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-18 11:16:39.031744


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-18 12:01:01.823877
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-18 12:16:30.926394


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-18 13:01:01.676642
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-18 13:16:50.107608


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-18 14:01:01.918903
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-18 14:16:22.545169


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-18 15:01:02.330526
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-18 15:16:49.431663


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-18 16:01:02.190937
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-18 16:16:33.264929


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-18 17:01:01.951329
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-18 17:16:23.095962


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-18 18:01:01.878953
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes1/2020-07-18/17
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-18 18:16:58.120763


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-18 19:01:01.910728
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-18 19:17:05.853682


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-18 20:01:01.545854
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes2/2020-07-18/19
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-18 20:17:15.890611


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-18 21:01:01.536590
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes2/2020-07-18/20
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-18 21:16:28.255740


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-18 22:01:02.026880
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-18 22:16:55.494340


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-18 23:01:02.179323
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-18 23:16:55.593890


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-19 00:01:02.333265
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-19 00:16:45.851543


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-19 01:01:01.791707
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-19 01:16:56.471522


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-19 02:01:02.166429
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-19 02:16:57.604181


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-19 03:01:02.296894
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-19 03:17:08.779194


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-19 04:01:02.524535
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-19 04:16:54.469273


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-19 05:01:02.261499
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-19 05:16:53.005139


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-19 06:01:01.760063
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-19 06:16:55.814652


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-19 07:01:01.594684
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-19 07:17:17.231760


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-19 08:01:02.075426
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-19 08:16:49.981111


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-19 09:01:01.781287
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-19 09:16:58.777564


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-19 10:01:01.496744
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes2/2020-07-19/09
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-19 10:16:53.224503


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-19 11:01:01.856247
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-19 11:22:25.498770


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-19 12:01:02.270474
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-19 12:16:52.178397


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-19 13:01:01.876887
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-19 13:16:50.625444


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-19 14:01:02.353920
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-19 14:17:01.856773


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-19 15:01:01.578521
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-19 15:16:45.359689


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-19 16:01:02.104767
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes2/2020-07-19/15
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-19 16:17:06.272865


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-19 17:01:01.918702
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-19 17:16:53.133929


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-19 18:01:01.918170
Now processing logs0
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes0/2020-07-19/17
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-19 18:17:16.891564


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-19 19:01:01.593703
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-19 19:17:10.610987


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-19 20:01:02.359152
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-19 20:17:05.278108


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-19 21:01:02.126596
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-19 21:17:02.828639


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-19 22:01:01.509422
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-19 22:16:46.057881


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-19 23:01:01.826271
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-19 23:17:11.920847


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-20 00:01:01.586298
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-20 00:17:19.170199


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-20 01:01:01.993756
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-20 01:17:12.826543


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-20 02:01:01.586184
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-20 02:17:00.478900


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-20 03:01:02.154053
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-20 03:16:56.405913


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-20 04:01:02.188077
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-20 04:17:20.134858


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-20 05:01:01.824288
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-20 05:17:49.503309


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-20 06:01:02.232410
Now processing logs0
making directory: /h
Now processing logs1
making directory: /h
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-20 06:17:44.172616


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-20 07:01:01.863582
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-20 07:17:17.653545


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-20 08:01:02.388009
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-20 08:16:45.374373


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-20 09:01:02.110765
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
Now processing logs2
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-20 09:16:42.824241


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
Execution of processAllLogs_new.py started at 2020-07-20 10:01:01.582898
Now processing logs0
making directory: /h
Now processing logs1
Could not process file: /home/ardelalegre/google-drive/ODAS/logs1/SST/cSST_2020-06-26_13:00:12_1.log
making directory: /h
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes1/2020-07-20/09
Now processing logs2
making directory: /home/ardelalegre/google-drive/ODAS/dataframes/dataframes2/2020-07-20/09
Now processing logs3
Execution of processAllLogs_new.py ended at 2020-07-20 10:22:09.124884


/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py:6201: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=True'.

To retain the current behavior and silence the warning, pass sort=False

  sort=sort)
